libnavigation-core/src/test/java/com/mapbox/navigation/NavigationControllerTest.kt

```diff
@@ -1,12 +1,15 @@
 package com.mapbox.navigation
 
+import android.content.Context
 import com.mapbox.android.core.location.LocationEngine
 import com.mapbox.android.core.location.LocationEngineRequest
 import com.mapbox.navigation.base.route.DirectionsSession
 import com.mapbox.navigation.navigator.MapboxNativeNavigator
-import com.mapbox.navigation.route.offboard.router.NavigationRoute
```

Comment:
This is the relevant change. We cannot explicitly depend on the offboard module, because it can be excluded.